query/src/main/scala/filodb/query/exec/SetOperatorExec.scala

```diff
@@ -118,8 +118,8 @@ final case class SetOperatorExec(queryContext: QueryContext,
 
     rhsRvs.foreach { rv =>
       val jk = joinKeys(rv.key)
-      // Don't add range vector if it is empty
-      if (jk.nonEmpty && !isEmpty(rv, rhsSchema)) {
```

Comment:
If RHS is empty, ``jk`` will be empty and nothing gets executed. What will happen to ``sum(foo{}) OR bar{}``  where ``bar{}`` is empty. The expectation is to have LHS results as [here](http://demo.robustperception.io:9090/graph?g0.expr=up%7B%7D%20OR%20bar%7B%7D&g0.tab=1&g0.stacked=0&g0.range_input=1h)

Reply:
Missed this one, the change is in AND only.